internal/config/config.go

```diff
@@ -204,15 +202,15 @@ func Load(fileName string) (*Configuration, error) {
 			Mode:  log.OutputText,
 		},
 	}
-
-	if err := viper.ReadInConfig(); err == nil {
-		if err := viper.Unmarshal(config); err != nil {
-			return nil, err
-		}
-	} else {
-		return nil, err
+	ctx := context.Background()
+	if err := viper.ReadInConfig(); err != nil {
+		log.Error(ctx, "error loading config file...", err)
 	}
 
+	if err := viper.Unmarshal(config); err != nil {
+		log.Error(ctx, "error unmurshalling config file", err)
```

Comment:
Typo: "unmarshalling"

BTW do you guys use VSCode? There is a great [spellchecker](https://github.com/streetsidesoftware/vscode-spell-checker) plugin.

Reply:
We are using Goland... maybe we can find something like that plugin

Reply:
Would this help? https://www.jetbrains.com/help/go/spellchecking.html